ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -61,6 +61,7 @@ RUN __LLVM_VERSION__="5.0" \
  && pip install --upgrade pip \
  && pip install --upgrade virtualenv \
  && pip3 install --upgrade pip \
```

Comment:
what if you do that before pip install --upgrade pip?

Reply:
Actually I don't think this solves the problem (`pip install --upgrade pip` says `Requirement already up-to-date: pip in /usr/local/lib/python2.7/dist-packages` and leaves `/usr/local/bin/pip` on Python 3.5).